backend/init_db/init.sql

```diff
@@ -0,0 +1,236 @@
+--======================================================--
+-- DROPS
+--======================================================--
+
+DROP TABLE IF EXISTS AuctionBooking CASCADE;
+DROP TABLE IF EXISTS AuctionTimeSlot CASCADE;
+DROP TABLE IF EXISTS StandardBooking CASCADE;
+DROP TABLE IF EXISTS TimeUnit CASCADE;
+DROP TABLE IF EXISTS Listing CASCADE;
+DROP TABLE IF EXISTS Booking CASCADE;
+DROP TABLE IF EXISTS ParkingSpot CASCADE;
+DROP TABLE IF EXISTS Car CASCADE;
+DROP TABLE IF EXISTS ResetPasswordToken CASCADE;
+DROP TABLE IF EXISTS ValidationToken CASCADE;
+DROP TABLE IF EXISTS SessionToken CASCADE;
+DROP TABLE IF EXISTS "User" CASCADE;
```

Comment:
why is there a double quote in User table name

Reply:
User is reserved in PostgreSQL, so in order to differentiate it, I unfortunately need to introduce the quotes.

Reply:
I would suggest coming up with another name for this table to avoid the double quote. What do you think of Users (with an s). Is that one reserved as well?

Reply:
Yes that would be a good idea as we would need to reference it with the double quotes EVERYWHERE. So it is a better idea to choose a different name.

Reply:
I agree.  I will update the table name to avoid the use of `"`.